src/material-examples/tree-nested-overview/tree-nested-overview-example.html

```diff
@@ -2,20 +2,20 @@
   <mat-tree-node *matTreeNodeDef="let node" matTreeNodeToggle>
     <li class="mat-tree-node">
       <button mat-icon-button disabled></button>
-      {{node.filename}}:  {{node.type}}
+      {{node.name}}
     </li>
   </mat-tree-node>
 
   <mat-nested-tree-node *matTreeNodeDef="let node; when: hasNestedChild">
     <li>
```

Comment:
It's not clear to me (or the user) why this example uses `<ul`> and `</li>` tags.

Reply:
you are right.  We should provide a better template in the following PR.